serendipity_event_markdown/serendipity_event_markdown.php

```diff
@@ -237,6 +251,49 @@ function _markdown_markup($text) {
         return Markdown($text);
     }
     */
+
+    function addToCSS(&$eventData) {
+        $eventData .= '
+/* Footnotes (generated by serendipity_event_markdown) */
+
+a.footnote-ref:after {
+  content: ")";
+}
+
+.footnotes hr {
+  border-top: dashed #cccccc;
+  border-width: 1px;
+}
+
+/* mostly taken from http://www.456bereastreet.com/archive/201105/styling_ordered_list_numbers/ */
+.footnotes ol {
+  counter-reset: li;
+  margin-top: 0.2em;
```

Comment:
Same here - I prefer the leading zero as I find it easier to read, but will change,